### PR TITLE
feat: Add zkEra adapter

### PR DESCRIPTION
**Pull request by zkeraFi**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
zkEra

##### Twitter Link:
https://x.com/zkerafi

##### List of audit links if any:
https://github.com/zkeraFi/contracts-core/blob/main/audits/zkEra%20Smart%20Contracts%20Finance%20Audit%20Report.pdf

##### Website Link:
https://zkera.fi

##### Logo (High resolution, will be shown with rounded borders):
https://zkera.fi/assets/zke.png

##### Current TVL:
$51,683 (ZLP + staked ZKE tokens), $31,259 (ZLP pool)

##### Chain:
TelosEVM

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
zkera-finance

##### Short Description (to be shown on DefiLlama):
zkEra Finance- Decentralized Perpetual Exchange on TelosEVM

##### Token address and ticker if any:
0xdf020cBd1897133978C7FcDF04B07E69d8934Efc

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Derivatives

##### Oracle Provider(s): Specify the oracle(s) used (e.g., Chainlink, Band, API3, TWAP, etc.):
DIA oracles

##### Implementation Details: Briefly describe how the oracle is integrated into your project:
zkEra contracts get prices for the calculations from the DIA pricefeeds

##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:
https://docs.zkera.fi/zkera-finance-overview - oracles section

##### forkedFrom (Does your project originate from another project):
GMX v1

##### methodology (what is being counted as tvl, how is tvl being calculated):
Deposited in the vault assets + staked tokens

##### Github org/user (Optional, if your code is open source, we can track activity):
zkeraFi

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/zkera exports TVL: 
        
 
 
    telos                     34.79 k
    
    total                    34.79 k

File: projects/zkera/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { gmxExports } = require("../helper/gmx");
+
+const vault = "0x17D3FdF3b017C96782dE322A286c03106C75C62E";
```
Comment (g1nt0ki) [inline projects/zkera/index.js:3]: but this is zkdx vault? https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/zkdx-finance/index.js#L17

Comment (ttbbio) [reply projects/zkera/index.js:3]: Our users try to add our info to defillama without our devs, so there is a conflict rn
you can check "Tvl Code"on the https://defillama.com/protocol/zkera-finance#information page, in this way you can see that this isn't zkdx project, it's zkEra listed by somebody else

Comment (zkeraFi) [reply projects/zkera/index.js:3]: Our users try to add our info to defillama without our devs, so there is a conflict rn
you can check "Tvl Code"on the https://defillama.com/protocol/zkera-finance#information page, in this way you can see that this isn't zkdx project, it's zkEra listed by somebody else

Comment (g1nt0ki) [reply projects/zkera/index.js:3]: okay, but if it is already called zkera on the app, why this PR?

Comment (g1nt0ki) [reply projects/zkera/index.js:3]: to fix the folder name?

Comment (zkeraFi) [reply projects/zkera/index.js:3]: We didn't know about other deployment and want to be in charge of any future updates, what should we do in this case?

**Pull request closed without merging**